### PR TITLE
Restructure Gallery

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: _data/examples.json [hunk 1]
```diff
@@ -193,37 +237,24 @@
       "title": "Trellis Area"
     }
   ],
-  "Layering": [
+  "Using Layer to Add Labels": [
```
Comment (domoritz) [inline _data/examples.json]: why capitalize add?

File: _data/examples.json [hunk 2]
```diff
@@ -134,15 +166,27 @@
...
+      "name": "rect_binned_heatmap",
+      "title": "Table Binned heatmap"
+    },
+    {
+      "name": "circle_github_punchcard",
+      "title": "Table Bubble Plot (Github Punch Card)"
     }
   ],
-  "Window Transform": [
+  "Advanced Calculations": [
```
Comment (domoritz) [inline _data/examples.json]: Let's leave this as window transforms until we have more.

File: _data/examples.json [hunk 3]
```diff
@@ -233,20 +264,45 @@
...
+      "title": "Candlestick Chart"
+    },
+    {
+      "name": "layer_histogram_global_mean",
+      "title": "Histogram with a Global Mean Overlay"
     },
     {
       "name": "layer_falkensee",
-      "title": "Falkensee",
+      "title": "Line chart with Hilighted Rectangles",
```
Comment (domoritz) [inline _data/examples.json]: typo

File: _data/examples.json (hunk 2 above)
Comment (kanitw) [reply _data/examples.json]: I think a lot of people who come to the gallery could be novice who do not know what window is.  Naming the section "window" makes it more confusing.

File: _data/examples.json (hunk 3 above)
Comment (kanitw) [reply _data/examples.json]: chart => Chart? 

(Falkensee is a terrible name as it doesn't suggest what this example is for at all.

File: _data/examples.json (hunk 1 above)
Comment (kanitw) [reply _data/examples.json]: It's a heading.

File: _data/examples.json (hunk 2 above)
Comment (kanitw) [reply _data/examples.json]: Plus, a lot of these examples are using window and calculate to achieve the result, not purely "window".

File: _data/examples.json (hunk 3 above)
Comment (domoritz) [reply _data/examples.json]: Hilighted

Comment (domoritz) [reply _data/examples.json]: > Falkensee is a terrible name

Also, thanks.

File: _data/examples.json (hunk 1 above)
Comment (domoritz) [reply _data/examples.json]: I think this heading is inconsistent with the others. The heading are about what, now what for.

Comment (kanitw) [reply _data/examples.json]: Well, using layer to label is different from when you really layer graphics.  Plus, some people will be looking around to see how to add labels.

**Pull request merged**